third_party/libwebrtc/webrtc/modules/congestion_controller/delay_based_bwe.cc

```diff
@@ -279,12 +279,16 @@ void DelayBasedBwe::IncomingPacketFeedback(
   }
 
   // [XZ 2019-10-21 added logging of per-pkt loss/delay, etc.]
-  RTC_CHECK_GT(packet_feedback.send_time_ms, 0);  // It is ensure by the caller
+  // RTC_CHECK_GT(packet_feedback.send_time_ms, 0);  // It is ensure by the caller
+
   // update delay info: d_fwd, d_base, d_queue, rtt
-  RTC_CHECK_LE(packet_feedback.send_time_ms, now_ms); // Make sure default_bwe_rtt_ms_ doesn't wrap
-  default_bwe_rtt_ms_ = now_ms - packet_feedback.send_time_ms;
-  RTC_CHECK_LE(packet_feedback.send_time_ms, packet_feedback.arrival_time_ms); // Make sure dtmp isn't negative
-  int64_t dtmp = packet_feedback.arrival_time_ms - packet_feedback.send_time_ms;
+  // RTC_CHECK_LE(packet_feedback.send_time_ms, now_ms); // Make sure default_bwe_rtt_ms_ doesn't wrap
+  // default_bwe_rtt_ms_ = now_ms - packet_feedback.send_time_ms;
+  default_bwe_rtt_ms_ = now_ms - packet_feedback.creation_time_ms;
+
+  // RTC_CHECK_LE(packet_feedback.send_time_ms, packet_feedback.arrival_time_ms); // Make sure dtmp isn't negative
+  // int64_t dtmp = packet_feedback.arrival_time_ms - packet_feedback.send_time_ms;
+  int64_t dtmp = packet_feedback.arrival_time_ms - packet_feedback.creation_time_ms;
```

Comment:
We still need to add RTC_CHECK_LE to make sure that the subtraction is >= 0 (note that the operation will be done as unsigned is both operands are unsigned, so it could still wrap)